packages/blade/src/components/Typography/BaseText/types.ts

```diff
@@ -42,6 +42,7 @@ export type BaseTextProps = {
   id?: string;
   color?:
     | BrandPrimaryColors
+    | 'brand.secondary.500'
```

Comment:
Lets expose all secondary shades like primary? they might need different shade at some place. Anyway we'll have to expose everything in future since brand refresh uses secondary colors a lot.

Reply:
@saurabhdaware Currently [we have only `500` in secondary.](https://blade.razorpay.com/?path=/story/guides-intro--page)

<img width="1229" alt="Screenshot 2023-10-25 at 12 40 39 PM" src="https://github.com/razorpay/blade/assets/46647141/5b05e0f7-f186-48cc-85c1-1308bbbd439f">

Reply:
Extract the type from the theme object instead of hardcoding it, otherwise we might forget to update later: 

```js
type BrandSecondaryColors = `brand.secondary.${keyof Theme['colors']['brand']['secondary']}`;
```

Reply:
@saurabhdaware @anuraghazra Done, refactored.